docs/dotnet-five.md

```diff
@@ -0,0 +1,148 @@
+
+## .NET release schedule
+
+A new major version of .NET is planned to release every November. .NET 5 will not have long-term support (LTS), however; .NET 6 in 2021 will be LTS. As major .NET versions ship, even-numbered releases will receive LTS. Minor releases will occur as needed.
+
+:::image type="content" source="media/dotnet-schedule.svg" alt-text=".NET release schedule":::
+
+## What happens to .NET Standard
+
+With .NET 5, .NET Standard is being de-emphasized. New application development can specify the `net5.0` target framework moniker (TFM) for all project types, including class libraries. Sharing code between .NET 5 workloads is simplified in that all you need is the `net5.0` TFM.
+
+However, if you're planning on sharing code between .NET Framework, .NET Core, and .NET 5 workloads - you can do so by specifying `netstandard2.0` as your TFM. For more information, see [How to specify target frameworks](standard/frameworks.md#how-to-specify-target-frameworks).
+
+## Language updates
+
+With .NET 5, the .NET programming languages are continuing to improve.
+
+### C# updates
+
+C# will move forward in lock-step with .NET releases. Developers writing .NET 5 apps will have access to the latest C# version and features. .NET 5 is paired with C# 9. C# 9 brings many new features to the language, here are a few highlights:
+
+- Records: Immutable reference types that behave like value types, and introduce the new `with` keyword into the language.
+- Relational pattern matching: Extends pattern matching capabilities to relational operators for comparative evaluations and expressions, including logical patterns - new keywords `and`, `or`, and `not`.
+- Top-level statements: As a means for accelerating adoption and learning of C#, the `Main` method can be omitted and application as simple as the following is valid:
+
+   ```csharp
+   System.Console.Write("Hello world!");
+   ```
+
+- Functional pointers: Language constructs that expose intermediate language (IL) the following opcodes `ldftn` and `calli`.
+
+For more information on the available C# 9 features, see [What's new in C# 9](csharp/whats-new/csharp-9.md).
+
+#### Source generators
+
+In addition to some of the highlighted new C# features, source generators are making their way into developer projects. Source generators allow code that runs during compilation to inspect your program and produce additional files that are compiled together with the rest of your code.
+
+For more information on source generators, see [Introducing C# source generators](https://devblogs.microsoft.com/dotnet/introducing-c-source-generators) and [C# source generator samples](https://devblogs.microsoft.com/dotnet/new-c-source-generator-samples).
+
+### F# updates
+
+F# is the .NET functional programming language, and with .NET 5, developers have access to F# 5. Here are several new features of F# 5, for more information, see [What's new in F# 5](fsharp/whats-new/fsharp-50.md).
```

Comment:
This would be clearer if "Here are several new features" is followed by the text about the features, and the "For more information" is found at the end of the features text.